cmd/kubeadm/app/cmd/phases/etcd.go

```diff
@@ -25,14 +25,30 @@ import (
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	etcdphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/etcd"
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
+	"k8s.io/kubernetes/pkg/util/normalizer"
+)
+
+var (
+	etcdLocalLongDesc = normalizer.LongDesc(`
+		Generates the static Pod manifest file for a local, single-node etcd instance and saves it to etcd.yaml file.
+		` + cmdutil.AlphaDisclaimer)
+
+	etcdLocalExample = normalizer.Examples(`
+		# Generates the static Pod manifest file for etcd, functionally 
+		# equivalent to what generated by kubeadm init.
+		kubeadm alpha phase etcd local
+
+		#  Generates the static Pod manifest file for etcd.
+		kubeadm alpha phase etcd local --config masterconfiguration.yaml
+		`)
 )
 
 // NewCmdEtcd return main command for Etcd phase
 func NewCmdEtcd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "etcd",
-		Short: "Generate static pod manifest file for etcd.",
-		RunE:  cmdutil.SubCmdRunE("etcd"),
+		Short: "Generates static Pod manifest file for etcd.",
```

Comment:
`Static` or `static` here?

Reply:
`static Pod`; according to the documentation style guide, `Pod` is an API objects and should be written in camel case like in the API, while `static` is an adjective and follows common grammar rules